app/models/cart.rb

```diff
@@ -19,6 +19,10 @@ def decrease_quantity_item(id)
     end
   end
 
+  def increase_quantity_item(id)
+    contents[id.to_s] = (contents[id.to_s] || 0) + 1
```

Comment:
Looks good, but do we need a zero case? Will the item show up if the count is zero?